spec-main/api-deprecate-spec.ts

```diff
@@ -7,6 +7,10 @@ describe('deprecate', () => {
     process.throwDeprecation = true
   })
 
+  after(() => {
```

Comment:
The test was failing as a side effect of this not being properly reset after the suite was run.

Reply:
Fantastic find. :rocket: I've no idea how you found this  

nit 1: since `process.throwDeprecation = true;` is set in `beforeEach()`, it would be slightly more correct to reset it in an `afterEach()`.

nit 2:  it would be more futureproof to store the current value in `beforeEach()` and restore _that_ value in `afterEach()` rather than unconditionally setting it to `false`

Reply:
necessary props to sam for assisting me on this find :D